### ISSUE TITLE
SQL: ALTER/DROP PROPERTY do not respect backtick quoting

### ISSUE THREAD

**Issue by theduke** [orientechnologies/orientdb#5276]
The SQL commands DROP PROPERTY and ALTER PROPERTY do not respect backtick quoting.

Example:

```
CREATE PROPERTY `tasks`.`my_property`
```
=> Works fine.

```
DROP PROPERTY `tasks`.`my_property` // => error: source class `tasks` not found
DROP PROPERTY tasks.`my_property` // => error: property `my_property` not found
// same thing with ALTER PROPERTY
```

**Comment by theduke** [comment]
Same thing with CREATE INDEX.

**Comment by luigidellaquila** [comment]
Fixed in develop branch (v 2.2.1)

Thanks

Luigi
